.gitignore

```diff
@@ -44,11 +44,13 @@ main.js
 main.js.map
 
 .idea
+.vscode
 npm-debug.log.*
 *.css.d.ts
 *.sass.d.ts
 *.scss.d.ts
 
+data
```

Comment:
I'd suggest `/data` and similarly for the following lines (avoid ignoring some random nested `data` file).